packages/format-currency/src/index.ts

```diff
@@ -223,10 +261,17 @@ export function createFormatter(): CurrencyFormatter {
 		defaultLocale = locale;
 	}
 
+	// If the user is inside the US using USD, they should only see `$` and not `US$`.
+	async function detectGeolocation() {
+		return setUsdCurrencySymbolBasedOnGeolocation();
```

Comment:
I don't see a good reason for the indirection - `setUsdCurrencySymbolBasedOnGeolocation() could be inlined here and we'd have one less defined function.